app/controllers/api/v1/events/nearests_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::Events::NearestsController < ApiController
+  def index
+    @events = Event.near(
+      [params[:lat], params[:lon]],
```

Comment:
We asked Brian to do this too, but you should put this in a private method called `coordinates` that returns these values. That way you can call Event.near like `Event.near(coordinates, params[:radius], units: :km` instead of having to deal with an multidimentional array.